crates/red_knot_python_semantic/resources/mdtest/type_properties/is_equivalent_to.md

```diff
@@ -0,0 +1,16 @@
+# Equivalence relation
+
```

Comment:
```suggestion

`is_equivalent_to` implements the equivalence relation for fully static types.

```

Reply:
It would also be good to link to https://typing.readthedocs.io/en/latest/spec/glossary.html#term-equivalent and/or https://typing.readthedocs.io/en/latest/spec/concepts.html#subtype-supertype-and-type-equivalence here